src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -315,6 +318,13 @@
   <Target Name="ExpandProjectReferences"
           DependsOnTargets="SplitProjectReferences">
 
+    <!-- Only rebuild project references when specified -->
```

Comment:
I'm fine with just Build for now, at some point we should consider if we also want to support Clean/Rebuild from these.